config/locales/forms/worldpay_forms/en.yml

```diff
@@ -4,8 +4,9 @@ en:
       failure:
         invalid_response: "Invalid WorldPay failure response"
         message:
-          REFUSED: "Your payment has been refused. Please try again or select a different payment method."
+          CANCELLED: "Your payment has been cancelled. Please try again or select a different payment method."
```

Comment:
Not a content bod, but should this read more along the lines

> Your attempt to pay was cancelled.

To my ears it currently sounds like payment was completed, and then subsequently cancelled. What do you 🤔 💭 ?

Reply:
Good point. Will change it now!

Reply:
@Cruikshanks Aaaand updated.